### PR TITLE
Add changes for end of 2016 leap second

### PR DESCRIPTION
**Pull request by jeanconn**
Add changes for end of 2016 leap second

### CODE REVIEW

**Comment by jeanconn** [comment]
This would be the fastest way to update using the hard coded leap seconds in the included XTime.   Future steps could include updates to use the proper library put together by Diab.

File: Chandra/test_Time.py [hunk 1]
```diff
@@ -186,6 +186,20 @@ def test_leapsec_2015(self):
         t2 = DateTime('2015-07-01 00:00:00').secs
         self.assertAlmostEqual(t2 - t1, 1.0)
 
+    def test_leapsec_2016(self):
+        """
+        Test that there are 4 clock ticks where one usually expects 3 (PR #15).
```
Comment (taldcroft) [inline Chandra/test_Time.py]: `PR #23`

File: setup.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
       description='Convert between various time formats relevant to Chandra',
       author_email='taldcroft@cfa.harvard.edu',
       py_modules=['Chandra.axTime3', 'Chandra.Time'],
-      version='3.19.1',
+      version='3.20.0',
```
Comment (taldcroft) [inline setup.py]: `3.20` not `3.20.0`

File: Chandra/test_Time.py (hunk 1 above)
Comment (taldcroft) [inline Chandra/test_Time.py]: I should have updated the test docstring in b25dc78058, but the docstring is not the whole story any more.  Add something like `and that initializing within the leap sec works as expected.`  (And put the PR# commend at the end).

Comment (jeanconn) [inline Chandra/test_Time.py]: Of the three setups, is it just the last one that you'd consider `initializing within the leap sec`?

Comment (jeanconn) [inline Chandra/test_Time.py]: I went with a docstring update and some comments.  Does that work for you?

**Pull request merged**